### PR TITLE
PXC-4082 Fix misspelling in versioning scheme (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/xtradb-cluster-versioning-scheme.mdmodified:   mkdocs-base.yml

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: docs/xtradb-cluster-version-numbers.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+</style>
+
+| 8.0.29| -21. | 2 |
+|---|---|---|
+| Base version | Minor | Patch |
+
+Percona uses semantic version numbering, which follows the pattern of base version, minor, and an optional patch fixes. Percona assigns unique, non-negative integers in increasing order for each version release. The version number combines the base Percona Server for MySQL version number and internal version numbers.
```
Comment (alina-derkach-oaza) [inline docs/xtradb-cluster-version-numbers.md]: an optional patch fixes -> optional patch fixes or an optional patch fix

File: docs/xtradb-cluster-version-numbers.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Understand version numbers
+
+A version number identifies the release and any updates. The product version denotes the latest stable version that contains the latest Generally Available (GA) features. You can compare the version of your Percona software with the latest release version to verify if you are up-to-date. When a version is released the contents of that version can not be modified. Any modifications are added to the next version.
```
Comment (alina-derkach-oaza) [inline docs/xtradb-cluster-version-numbers.md]: When a version is released -> add comma

File: docs/xtradb-cluster-version-numbers.md (hunk 1 above)
Comment (patrickbirch) [reply docs/xtradb-cluster-version-numbers.md]: rewritten

File: docs/xtradb-cluster-version-numbers.md (hunk 2 above)
Comment (patrickbirch) [reply docs/xtradb-cluster-version-numbers.md]: rewritten

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/xtradb-cluster-version-numbers.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+|---|---|---|
+| Base version | minor build | custom build |
+
+Percona uses semantic version numbering, which follows the pattern of base version, minor build, and an optional custom build. Percona assigns unique, non-negative integers in increasing order for each minor build release. The version number combines the base Percona Server for MySQL version number, the minor build version, and the custom build version, if needed.
+
+The version numbers for Percona XtraDB Cluster 8.0.20-11.2 define the following information:
+
+* Base version - the leftmost numbers indicate the Percona Server for MySQL version used as a base. An increase in base version resets the minor build version and the custom build version to 0. 
+
+* minor build version - an internal number that denotes which version that increases by one then the custom build fix is reset to 0.
```
Comment (alina-derkach-oaza) [inline docs/xtradb-cluster-version-numbers.md]: an internal number that denotes which version that increases - this sentence is incomplete

File: docs/xtradb-cluster-version-numbers.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        text-align: center;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+</style>
+
+| 8.0.20| -11. | 2 |
+|---|---|---|
+| Base version | minor build | custom build |
```
Comment (alina-derkach-oaza) [inline docs/xtradb-cluster-version-numbers.md]: Minor build
Custom build

File: docs/xtradb-cluster-version-numbers.md (hunk 3 above)
Comment (alina-derkach-oaza) [inline docs/xtradb-cluster-version-numbers.md]: Minor build version

File: docs/xtradb-cluster-version-numbers.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Percona uses semantic version numbering, which follows the pattern of base version, minor build, and an optional custom build. Percona assigns unique, non-negative integers in increasing order for each minor build release. The version number combines the base Percona Server for MySQL version number, the minor build version, and the custom build version, if needed.
+
+The version numbers for Percona XtraDB Cluster 8.0.20-11.2 define the following information:
+
+* Base version - the leftmost numbers indicate the Percona Server for MySQL version used as a base. An increase in base version resets the minor build version and the custom build version to 0. 
+
+* minor build version - an internal number that denotes which version that increases by one then the custom build fix is reset to 0.
+
+* custom build version - an optional number assigned to custom builds used for bug fixes. The software features, unless they're included in the bug fix, don't change.
```
Comment (alina-derkach-oaza) [inline docs/xtradb-cluster-version-numbers.md]: Custom build version

File: docs/xtradb-cluster-version-numbers.md (hunk 4 above)
Comment (patrickbirch) [reply docs/xtradb-cluster-version-numbers.md]: Capitalized.

File: docs/xtradb-cluster-version-numbers.md (hunk 3 above)
Comment (patrickbirch) [reply docs/xtradb-cluster-version-numbers.md]: fixed.

File: docs/xtradb-cluster-version-numbers.md (hunk 5 above)
Comment (patrickbirch) [reply docs/xtradb-cluster-version-numbers.md]: rewritten

File: docs/xtradb-cluster-version-numbers.md (hunk 3 above)
Comment (patrickbirch) [reply docs/xtradb-cluster-version-numbers.md]: Capitalized.

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Pull request merged**